sprokit/conf/dashboard.ctest.in

```diff
@@ -60,9 +60,9 @@ if (NOT TYPE)
   set(TYPE "Experimental")
 endif ()
 
-if ("$Format:$" STREQUAL "")
+if ("" STREQUAL "")
   set(UPDATE FALSE)
-  set(BRANCH "$Format:%d$")
+  set(BRANCH " (dev/uncrustify/arrows/ffmpeg)")
```

Comment:
Looks good except for this; not sure how this got in here...

Reply:
Fixed